.github/workflows/tests.yml

```diff
@@ -30,12 +30,12 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: nvim-lua/plenary.nvim
-          path: plenary.nvim
+          path: tests/.deps/plenary
```

Comment:
These paths are not correct. CI still clones the deps instead of using this one because it should be nested inside `plugins` directory (https://github.com/nvim-orgmode/orgmode/actions/runs/6313652906/job/17142196496?pr=616#step:7:8). 

Since we fetch the deps with the test script now, I think we can just remove these two steps and let the script clone everything that it needs.

Reply:
Sounds good, I'll remove them then.

Reply:
Done.